lang/pt_BR.json

```
@@ -1,11 +1,169 @@
 {
+    ":Resource could not be changed to :value": ":Resource não pôde ser mudado para :value",
+    ":Resource could not be created": ":Resource não pôde ser criado",
+    ":Resource could not be deleted": ":Resource não pôde ser excluído",
+    ":Resource could not be restored": ":Resource não pôde ser restaurado",
+    ":Resource could not be updated": ":Resource não pôde ser atualizado",
+    ":Resource created successfully": ":Resource criado com sucesso",
+    ":Resource deleted successfully": ":Resource excluído com sucesso",
+    ":Resource restored successfully": ":Resource restaurado com sucesso",
+    ":Resource updated successfully to :value": ":Resource atualizado com sucesso para :value",
+    ":Resource updated successfully": ":Resource atualizado com sucesso",
+    ":firstItem - :lastItem of :total :resource": ":firstItem - :lastItem de :total :resource",
+    "About Me": "Sobre Mim",
+    "About": "Sobre",
+    "Active Players": "Jogadores Ativos",
+    "Activity": "Atividade",
+    "Appearance": "Aparência",
+    "Architect": "Arquiteto",
+    "Average Completion": "Média de Conclusão",
+    "Avg. Completion": "Média de Conclusão",
+    "Blue": "Azul",
+    "Color": "Cor",
+    "Community": "Comunidade",
+    "Connect": "Conecte-se",
+    "Country": "País",
+    "Currently Online": "Online no momento",
+    "Default": "Padrão",
+    "Dark": "Escuro",
+    "Date/Time Format": "Formato de Data/Hora",
+    "Developers Code of Conduct": "Código de Conduta dos Desenvolvedores",
+    "Display Preferences": "Preferências de Tela",
+    "Documentation": "Documentação",
     "Edit": "Editar",
-    "Save": "Salvar",
+    "Founder": "Fundador",
+    "Green": "Verde",
+    "Inbox": "Caixa de Entrada",
+    "Language": "Linguagem",
+    "Legal": "Legal",
```

```suggestion
    "Legal": "Legalidade",
```

This appears to be a mistranslation.